### PR TITLE
Deploy new calcom version 29 08 23

### PR DESCRIPTION
**Pull request by pavanas**
## Description

Deploying new version for Calcom in Staging 😬 

## Jira task
-

## Type of change

feat: new deploy + change runners

## How to verify the changes:

-

## Screenshots (For all UI changes):

Put your screenshots here...

### CODE REVIEW

**Review by ahinchliff — APPROVED** [review]
🚀

File: .github/workflows/build-and-push-to-ecr-api.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
```
Comment (reecefenwick) [inline .github/workflows/build-and-push-to-ecr-api.yml:21]: We should only use the larger runners where we are seeking performance improvements, usually for more intensive workflows that run test suites

See related discussion https://github.com/MontuGroup/pms-service/pull/715#issuecomment-1695085803

TLDR; is the larger runners are a shared resource, so it's best to be conservative with where we use it

**Review by webfella — APPROVED** [review]
Just took a look at the build tasks and they seem intensive enough to bump to the larger runner, at least for the short term while we're iterating a bunch to release this feature.

@reecefenwick brings up a good point though so maybe we revisit these once it's stable in prod.

File: .github/workflows/build-and-push-to-ecr-api.yml (hunk 1 above)
Comment (pavanas) [reply .github/workflows/build-and-push-to-ecr-api.yml:21]: I agree! we did this change as Connor mentioned to me that in his last deploy things were getting > 20min to finalize :) We'll see how it goes this time!
